src/products.js

```diff
@@ -0,0 +1,13 @@
+import items from './data/items.js';
+import renderItem from '/src/render-item.js';
+import store from '../src/data/store.js';
+
+const list = document.getElementById('items');
+
+for(let i = 0; i < items.length; i++) {
```

Comment:
instead of items here, you should be pulling your products list from local storage.

const products = store.getProducts();

so instead of pulling from your local files, it will pull from localstorage and update when you add items with your form.